### ISSUE TITLE
Add intervention parameter to the run_sim RPC

### ISSUE THREAD

**Issue by sayers24** [InstituteforDiseaseModeling/covasim#75]
Support for the cova_app.py to accept the necessary parameters for the intervention data

**Comment by sayers24** [comment]
@lgeorge-idm, it will help to collaborate on this one. If the run_sim request can accept the H/S/W/C matrix, the web app could have the flexibility to construct any combination of interventions, correct?

**Comment by cliffckerr** [comment]
@sayers24 see https://github.com/InstituteforDiseaseModeling/covasim/issues/72#issuecomment-610657051, you can just pass it in via the pars dict (although NB, you'll have to use this to jsonify interventions: https://github.com/InstituteforDiseaseModeling/covasim/blob/master/covasim/base.py#L236)

**Comment by sayers24** [comment]
Awesome, Thanks @cliffckerr!

**Comment by lgeorge-idm** [comment]
Closed by #71

**Comment by lgeorge-idm** [comment]
@cliffckerr , working on this now and I'm running into a few issues:
1. There doesn't seem to be any way to end contact tracing
2. There isn't anyway to differentiate children if you have overlapping social distancing measures with school closures (assumption based on the [Unified UI doc](https://docs.google.com/document/d/1Q3-cLk9lo8Oizgs4JZTnqDCBjWu3oE93S42B9rSinfY/edit#heading=h.n55nl753eyhd))
3. Minor(?): the `Intervention.to_json()` doesn't serialize numpy types correctly for use with `json.dump`

Here's a first pass at building out the interventions list based on the following campaign:
![newplot](https://user-images.githubusercontent.com/43424134/79024841-fced0c00-7b38-11ea-8bc1-7bf939bab182.png)
The request parameters for the interventions:
```json
{
    "social_distance": [
        {
            "start": 1,
            "end": 45,
            "level": "aggressive"
        }
    ],
    "school_closures": [
        {
            "start": 25,
            "end": 75
        }
    ],
    "symptomatic_testing": [
        {
            "start": 1,
            "end": 45,
            "level": 60
        },
        {
            "start": 46,
            "end": 90,
            "level": 90
        }
    ],
    "contact_tracing": [
        {
            "start": 1,
            "end": 23
        }
    ]
}
```


Attempt 1 at building out the interventions list using the cv.interventions.* methods:
```python
[
    {
        # Start Social Distancing - aggressive
        'results': {},
        'days': array([1, 1, 1, 1]),
        'changes': array([1.5, 0., 0.1, 0.1]),
        'layers': ['h', 's', 'w', 'c'],
        'orig_betas': None,
        'InterventionType': 'change_beta'
    },
    {
        # End Social Distancing - aggressive
        'results': {},
        'days': array([45, 45, 45, 45]),
        'changes': array([1, 1, 1, 1]),
        'layers': ['h', 's', 'w', 'c'],
        'orig_betas': None,
        'InterventionType': 'change_beta'
    },
    {
        # Start School Closures
        'results': {},
        'days': array([25, 25, 25, 25]),
        'changes': array([1.5, 0., 0.7, 1.]),
        'layers': ['h', 's', 'w', 'c'],
        'orig_betas': None,
        'InterventionType': 'change_beta'
    },
     {
        # End School Closures
        'results': {},
        'days': array([75, 75, 75, 75]),
        'changes': array([1, 1, 1, 1]),
        'layers': ['h', 's', 'w', 'c'],
        'orig_betas': None,
        'InterventionType': 'change_beta'
    },
    {
        # Start Symptomatic Testing @ 60%
        'results': {},
        'symptomatic_prob': 1,
        'asymptomatic_prob': 0,
        'quarantine_prob': 0,
        'symp_quar_prob': 0,
        'test_sensitivity': 0.6,
        'loss_prob': 0.0,
        'test_delay': 1,
        'start_day': 1,
        'InterventionType': 'test_prob'
    },
    {
        # End Symptomatic Testing @ 60% 
        # Start Symptomatic Testing @ 90%
        'results': {},
        'symptomatic_prob': 1,
        'asymptomatic_prob': 0,
        'quarantine_prob': 0,
        'symp_quar_prob': 0,
        'test_sensitivity': 0.9,
        'loss_prob': 0.0,
        'test_delay': 1,
        'start_day': 45,
        'InterventionType': 'test_prob'
    },
    {
        # Start Contact Tracing
        'results': {},
        'trace_probs': {
            'h': 1.0,
            's': 0.8,
            'w': 0.7,
            'c': 0.3
        },
        'trace_time': {
            'c': 7
        },
        'contact_reduction': None,
        'start_day': 1,
        'InterventionType': 'contact_tracing'
    }
]
```

I haven't even attempted to run this yet because I'm 99.9% sure it won't model as expected. I know how to construct the matrix as it _should_ be, but I don't have enough granularity to actually do that, at least not with contact layers, which seems important.

**Comment by cliffckerr** [comment]
Closed by #171
